bridge/deployments/development/.chainId

```diff
@@ -0,0 +1 @@
+1337
```

Comment:
deployments/development should be in .gitignore as this will change every time someone uses ganache.

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/267/commits/d776031b3a42b2933cbc75d0c9c920c3a3739f42